### PR TITLE
MINOR: improve logging of consumer system tests

### PR DESCRIPTION
**Pull request by hachikuji**

### CODE REVIEW

File: tests/kafkatest/services/verifiable_consumer.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ class VerifiableConsumer(BackgroundThreadService):
     logs = {
         "verifiable_consumer_stdout": {
             "path": STDOUT_CAPTURE,
-            "collect_default": False},
```
Comment (ewencp) [inline tests/kafkatest/services/verifiable_consumer.py:126]: We probably shouldn't change this default to true -- we should override it in the specific test we need by calling `self.mark_for_collect(consumer, 'verifiable_consumer_stdout')` where `consumer` is the `VerifiableConsumer` instance. stdout for verifiable consumer can result in _huge_ log files, so collecting them by default will result in very large archived data for some tests.

Comment (hachikuji) [inline tests/kafkatest/services/verifiable_consumer.py:126]: It shouldn't be any bigger than the consumer's debug log (I think), but perhaps we only need it in tests which specifically target the consumer's behavior.

Comment (ewencp) [inline tests/kafkatest/services/verifiable_consumer.py:126]: Verifiable consumer necessarily prints something for every record right? I would think that could easily grow larger than the debug log. But in any case it doesn't look like anything is collected by default for VerifiableConsumer, so I'm not sure why any other log file size would be relevant for comparison...

Comment (hachikuji) [inline tests/kafkatest/services/verifiable_consumer.py:126]: We only print the start and end offsets of message batches received at the moment. But eventually we'll probably have test cases which require seeing message payloads.

**Comment by hachikuji** [comment]
@ewencp I turned off stdout collection since I can probably get by with the event logging itself. Note that we may may have to turn down that logging (or at least choose the events we log more selectively) if VerifiableConsumer ever does log messages individually. This change results in about 5-10 Mb increase in overall retained data.

**Comment by ewencp** [comment]
@hachikuji Why not collect it in just the one test you care about as I suggested? If you need the info, I don't have a problem collecting it, I just thought it made sense to limit extra collection of data to the specific test we need it for.

I think the confusion re: verbosity came up because I know we have a verbose mode in VerifiableConsumer, but I guess we never enable it in the system tests...

**Comment by hachikuji** [comment]
@ewencp I care about one test at the moment, but I'm assuming that others may also have transient failures in the future. The problem is that these failures tend to be hard to reproduce (e.g. I still haven't reproduced the one in 3513 yet), so I need to get as much information from each failure as possible.

**Comment by ewencp** [comment]
LGTM

**Pull request closed without merging**